docs/earthly-command/earthly-command.md

```diff
@@ -961,3 +961,48 @@ If you are logged into the CLI the url will contain a token used to link your OA
 * ```
   earthly web --provider=github
   ```
+
+## earthly docker-build
+
+#### Synopsis
+
+*
+  ```
+  earthly [options] docker-build [--dockerfile <dockerfile-path>] [--tag=<image-tag>] [--target=<target-name>] [--platform <platform>] <build-context-dir> [--arg1=arg-value] 
+  ```
+
+#### Description
+
+The command `earthly docker-build` builds a docker image from a Dockerfile instead of an Earthfile.
+The <context-path> is the path where the Dockerfile build context exists. By default, it is assumed that a file named Dockerfile exists in that directory.
+
+#### Options
```

Comment:
I think it's ok to mention it as part of the description

Reply:
Oh, right. I didn't see that in the description.

Minor note: line `977` will be part of the same paragraph as `976`, because markdown ignores single newlines. I just wanted to check that that's what you want. Otherwise :+1:

Reply:
good eye! updated